### ISSUE TITLE
cross-validation?

### ISSUE THREAD

**Issue by shlid007** [nliulab/AutoScore#8]
Hi - is cross-validation possible instead of using a separate validation set (without having to modify the core delivered code)? Or what about bootstrapping?

**Comment by fengx13** [comment]
Our tutorial has covered cross-validation part. You may refer to https://nliulab.github.io/AutoScore/04-autoscore.html#demo2

**Comment by shlid007** [comment]
Thank you Dr. Feng! To do that on the final model, not just training, I believe Dr. Liu said to split my test dataset into x partitions and do CV that way, correct? Example: to do 5-fold CV, split test dataset into five portions and run the model against each portion (from my test set).

Reason for doing CV on a test set is, I want to collect multiple observations on the final model to do statistical testing of differences (across, say, logistic regression, random forest, and AutoScore).

Hopefully my explanation makes sense ... If I interpreted the approach incorrectly, please let me know.

Thanks,
Sue Lhymn
________________________________
